### PR TITLE
136 Create Login form

### LINKED ISSUE #136: Create Sign in form and event handlers
Create Sign in form and event handlers for Signin Component. This should grabs the user input {name, surname, password} and passes them as an {user} object, as an argument for the signin action. #

### PR DESCRIPTION
**Pull request by rebeccazhangnz**
created sigin form
closes #136

### CODE REVIEW

File: client/components/Login.jsx [hunk 1]
```diff
@@ -1,20 +1,59 @@
...
+      name: '',
+      surname: '',
+      password: '' })
+    e.preventDefault()
+  }
+
+  render () {
+    if (this.props.auth.loggedIn) {
+      return <Redirect to='/' />
+    }
```
Comment (don-smith) [inline client/components/Login.jsx:38]: I wonder if this conditional would be better placed inside of `componentDidMount`?

File: client/components/Login.jsx [hunk 2]
```diff
@@ -1,20 +1,59 @@
 import React from 'react'
+import { Redirect } from 'react-router-dom'
+import { connect } from 'react-redux'
+import { signin } from '../actions/auth'
```
Comment (bryceknz) [inline client/components/Login.jsx:4]: Please name this 'login' to be consistent with the rest of the code base.

**Review by don-smith — APPROVED** [review]
Try that conditional in `componentDidMount` if it works, keep it there. Otherwise, keep it where it is. Thanks!

File: client/components/Login.jsx [hunk 3]
```diff
@@ -1,20 +1,59 @@
...
+    this.setState({
+      name: '',
+      surname: '',
+      password: '' })
+    e.preventDefault()
+  }
+
+  render () {
+    if (this.props.auth.loggedIn) {
+      return <Redirect to='/' />
```
Comment (emilyparkes) [inline client/components/Login.jsx:37]: this will be to their dashboard at `student/${:id}`

**Review by emilyparkes — CHANGES_REQUESTED** [review]

**Review by emilyparkes — APPROVED** [review]

**Pull request merged**